src/models/connect.ts

```diff
@@ -166,7 +166,7 @@ export default class Connect {
     if (!auth.clientId) {
       auth.clientId = this.clientId;
     }
-    if (auth.hasOwnProperty('scopes')) {
+    if (auth.hasOwnProperty('provider') && (auth as any).provider != 'nylas') {
```

Comment:
What's the meaning behind making sure the provider isn't nylas here? When might this come up?

Reply:
@philrenaud I've received some feedback that this would be a nice to have given that some of our customers who migrated to v6.0 have missed this change of omitting provider.